emscripten.py

```diff
@@ -178,6 +179,8 @@ def compile_javascript(symbols_only=False):
     stderr_file = os.path.abspath(stderr_file)
     logger.info('logging stderr in js compiler phase into %s' % stderr_file)
     stderr_file = open(stderr_file, 'w')
+  else:
+    stderr_file = subprocess.PIPE
```

Comment:
iiuc this stashes stderr so you can read it below, but I don't see you print it out later, so that means stderr is no longer visible to the user? There are warnings that come from the JS compiler that we need to show iirc.